app/src/main/java/protect/card_locker/LoyaltyCard.java

```diff
@@ -34,12 +34,14 @@ public class LoyaltyCard implements Parcelable {
     public final long lastUsed;
     public int zoomLevel;
 
+    public int usage_number;
+
     public LoyaltyCard(final int id, final String store, final String note, final Date validFrom,
                        final Date expiry, final BigDecimal balance, final Currency balanceType,
-                       final String cardId, @Nullable final String barcodeId,
-                       @Nullable final CatimaBarcode barcodeType,
+                       final String cardId, @Nullable final String barcodeId, @Nullable final CatimaBarcode barcodeType,
                        @Nullable final Integer headerColor, final int starStatus,
-                       final long lastUsed, final int zoomLevel, final int archiveStatus) {
+                       final long lastUsed, final int zoomLevel, final int archiveStatus,
+                       int usage_number) {
```

Comment:
Can't this be final like the rest?

Also, please use camel-case like everywhere else.